docs/guides/registration/badge_designer.md

```diff
@@ -0,0 +1,100 @@
+---
+sidebar_position: 3
+---
+
+# Badge Designer
+
+<splitColumn>
+  <div>
+    <p>
+      The Badge Designer allows you to customize the badge layout and styling, using a layer style positioning of elements, to enable compatibility with a wide array of printing hardware and design needs.
+    </p>
+    <p>
+      You can select from a variety of user-generated content, such as real name, badge ID, badge name, and attendance tier.
+    </p>
+  </div>
+  <div>
+    <img src="/img/tutorial/badgeDesigner.png" />
+  </div>
+</splitColumn>
+
+:::info
+Using the Badge Designer currently requires the `system:settings:update` and `system:badgedesign:read` permission. Creating / updating badge designs additionally requires the `system:badgedesign:update` permission.
```

Comment:
It might be good to mention permissions in other contexts as well.

Reply:
Will make another ticket on this and do it in a separate PR.